holoviews/core/dimension.py

```diff
@@ -703,6 +710,34 @@ def map(self, map_fn, specs=None, clone=True):
         else:
             return map_fn(self) if applies else self
 
+    def transform(self, *args, drop=False, drop_duplicate_data=True, **kwargs):
```

Comment:
-1 on adding this, we can now do this using `.apply`. I'll therefore suggest we add `.apply.transform`.

Reply:
Ok by me.

Reply:
Sounds good!